executor/adapter.go

```diff
@@ -243,7 +243,7 @@ func (a *ExecStmt) Exec(ctx context.Context) (sqlexec.RecordSet, error) {
 
 	var txnStartTS uint64
 	if sctx.Txn(false).Valid() {
-		txnStartTS = sctx.Txn().StartTS()
+		txnStartTS = sctx.Txn(true).StartTS()
```

Comment:
@crazycs520 Sorry, I delete your comment mistakenly,  because `sctx.Txn(false).Valid()` is just check the txn is valid, not need to active it there.

Reply:
@crazycs520 is right, it's better to use `false` here.
Never mind, use `true` here doesn't cause trouble, because when `sctx.Txn(false).Valid()` is true,
Txn() status is not pending.

```
	if s.txn.pending() && active {
           // active ....
    } else {
           return txn  // 
    }
```